src/app/item-popup/PlugTooltip.m.scss

```
@@ -1,6 +1,19 @@
+@import '../variables';
```

Super minor note - we swapped to a newer Sass lib, so this should be:
```suggestion
@use '../variables' as *; 
```